example/odoo/migration.yml

```diff
@@ -1,6 +1,6 @@
 migration:
   options:
-    install_command: odoo
+    install_command: odoo --without-demo=${WITHOUT_DEMO:-True}
```

Comment:
that is compatible with Version > 8 ?

Reply:
@Tonow-c2c yes it is

@p-tombez it triggers some memories to me... `--without-demo` buggy in the config file... something to document in the 'legacy section' of the docs as well :)